chaoscenter/graphql/server/graph/model/models_gen.go

```diff
@@ -1871,6 +1871,8 @@ type RegisterInfraRequest struct {
 	Tolerations []*Toleration `json:"tolerations,omitempty"`
 	// Tags of the infra
 	Tags []string `json:"tags,omitempty"`
+	// Installation type indicating if the infra uses helm or kubernetes manifest
+	InstallationType *string `json:"installationType,omitempty"`
```

Comment:
There are changes in the models_gen.go, but no changes in the .graphqls file. How are these changes being generated?